### PR TITLE
:seedling: add tracker API Test

### PR DESCRIPTION
**Pull request by khareyash05**
Added a Tracker API Test

### CODE REVIEW

File: test/api/tracker/api_test.go [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+			projectsList, err := Tracker.ListProjects(r.ID)
+			if err != nil {
+				// check for type of service (maybe Connected?)
+				// if API service then print Not connected to Jira and pass the test
+				// else fail the test
+				if r.Connected == false {
+					t.Logf("Not connected to Jira(Thus passing the API Test)")
+				}
+				if r.Connected == true {
```
Comment (aufi) [inline test/api/tracker/api_test.go:81]: Thinking why not use ```} else {``` here (or do you want extent the condition with more checks?)

File: test/api/tracker/api_test.go [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+	if got.URL != expected.URL {
+		t.Errorf("Different Url Got %v, expected %v", got.URL, expected.URL)
+	}
+	if got.Kind != expected.Kind {
+		t.Errorf("Different Kind Got %v, expected %v", got.Kind, expected.Kind)
+	}
+	if got.Connected != expected.Connected {
+		t.Errorf("Different Connected Got %v, expected %v", got.Connected, expected.Connected)
+	}
+	if got.LastUpdated != expected.LastUpdated {
```
Comment (aufi) [inline test/api/tracker/api_test.go:183]: Just a note on this field, I'd prefer not include LastUpdated field in Equality check (it is similar to e.g. CreatedAt which we don't assert neither).

**Review by aufi — COMMENTED** [review]
Added few notes, but overall looks good!

**Comment by aufi** [comment]
Related to https://github.com/konveyor/operator/issues/220

File: test/api/tracker/api_test.go (hunk 1 above)
Comment (khareyash05) [reply test/api/tracker/api_test.go:81]: No I was thinking if this condition would suffice or if we would need more conditions later. Will update.Thanx

File: test/api/tracker/api_test.go (hunk 2 above)
Comment (khareyash05) [reply test/api/tracker/api_test.go:183]: Will update

**Review by aufi — APPROVED** [review]
LGTM

**Pull request merged**